gradle-conventions/settings.gradle.kts

```diff
@@ -2,16 +2,28 @@ rootProject.name = "gradle-conventions"
 
 pluginManagement {
     repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
         gradlePluginPortal()
         mavenCentral()
-        google()
     }
 }
 
 dependencyResolutionManagement {
     repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
         mavenCentral()
-        google()
         gradlePluginPortal()
```

Comment:
Since `gradle-conventions` is for Gradle plugins, it seems to have libraries that reference `gradlePluginPortal()` like `com.mikepenz.aboutlibraries.plugin.gradle.plugin`. 
So I'm keeping the `gradlePluginPortal()` specification here.